Use pretty colors in the output?

https://github.com/chalk/chalk can do it for you, though if you want just an angry red & bold color it's easy to do inline.

I'd be willing to send a PR too!

In a little more detail: this is what I get from putting it in a nested `package.json` script:

```javascript
    // ...
    "setup": "npm run setup:engines && npm run setup:script",
    "setup:engines": "check-node-version --node 9",
    "setup:script": "node scripts/setup.js",
    // ...
```

```text
C:\Code\MyRepo (bugfix/hard-package-versions)
λ npm run setup

> electric-boogaloo@0.1.0 setup C:\Code\MyRepo
> npm run setup:engines && npm run setup:script

npm WARN invalid config loglevel="notice"

> electric-boogaloo@0.1.0 setup:engines C:\Code\MyRepo
> check-node-version --node 9

node: 8.8.1
Error: Wanted node version 9 (>=9.0.0 <10.0.0)
To install node, run `nvm install 9` or see https://nodejs.org/

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Code\\MyRepo\\node_modules\\npm\\bin\\npm-cli.js" "run" "setup:engines"
npm ERR! node v8.8.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! electric-boogaloo@0.1.0 setup:engines: `check-node-version --node 9`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the electric-boogaloo@0.1.0 setup:engines script 'check-node-version --node 9'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the electric-boogaloo package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     check-node-version --node 9
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs electric-boogaloo
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls electric-boogaloo
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Code\MyRepo\npm-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! electric-boogaloo@0.1.0 setup: `npm run setup:engines && npm run setup:script`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the electric-boogaloo@0.1.0 setup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\jogol\AppData\Roaming\npm-cache\_logs\2017-11-29T00_29_33_449Z-debug.log
```

Someone with a very old Node version might not see the actual error. In fact, that happened earlier today with a teammate of mine.

--- a/bin.js
+++ b/bin.js
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 "use strict";
 
+var chalk = require("chalk");
 var minimist = require("minimist");
 var check = require("./");
 var PROGRAMS = require("./").PROGRAMS;
@@ -9,22 +10,22 @@ var path = require("path");
 
 function logVersionError(name, err) {
   if (err.stderr) {
-    console.error(err.stderr);
+    console.error(chalk.red.bold(err.stderr));
   }
   else if (err.execError) {
-    console.error(err.execError.message);
+    console.error(chalk.red.bold(err.execError.message));
   }
   else {
-    console.error(err.message);
+    console.error(chalk.red.bold(err.message));
   }
 }
 
 function printInstalledVersion(name, info) {
   if (info.version) {
-    console.log(name + ": " + info.version);
+    console.log(chalk.grey(name + ": ") + chalk.white(info.version));
   }
   if (info.notfound) {
-    console.error(name + ': not installed');
+    console.error(chalk.red.bold(name + ': not installed'));
   }
   if (info.error) {
     logVersionError(name, info.error);
@@ -43,8 +44,8 @@ function printVersions(result, print) {
     if (!isSatisfied) {
       var raw = info.wanted.raw;
       var range = info.wanted.range;
-      console.log("Error: Wanted " + name + " version " + raw + " (" + range +")");
-      console.log(PROGRAMS[name].getInstallInstructions(raw));
+      console.error(chalk.red.bold("Error: Wanted " + name + " version " + raw + " (" + range +")"));
+      console.log(chalk.red(PROGRAMS[name].getInstallInstructions(raw)));
     }
   });
 }

--- a/bin.js
+++ b/bin.js
@@ -45,7 +45,7 @@ function printVersions(result, print) {
       var raw = info.wanted.raw;
       var range = info.wanted.range;
       console.error(chalk.red.bold("Error: Wanted " + name + " version " + raw + " (" + range +")"));
-      console.log(chalk.red(PROGRAMS[name].getInstallInstructions(raw)));
+      console.log(chalk.red.bold(PROGRAMS[name].getInstallInstructions(raw)));
     }
   });
 }

--- a/bin.js
+++ b/bin.js
@@ -22,7 +22,7 @@ function logVersionError(name, err) {
 
 function printInstalledVersion(name, info) {
   if (info.version) {
-    console.log(chalk.grey(name + ": ") + chalk.white(info.version));
+    console.log(name + ": " + chalk.bold(info.version));
   }
   if (info.notfound) {
     console.error(chalk.red.bold(name + ': not installed'));

--- a/package.json
+++ b/package.json
@@ -24,6 +24,7 @@
   },
   "homepage": "https://github.com/parshap/check-node-version#readme",
   "dependencies": {
+    "chalk": "^2.3.0",
     "map-values": "^1.0.1",
     "minimist": "^1.2.0",
     "object-filter": "^1.0.2",

